docs/diagrams/HelpCommandParsingSequenceDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+!include style.puml
+
+box UI UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+participant ":HelpWindow" as HelpWindow UI_COLOR
+end box
+
+box Logic LOGIC_COLOR_T1
+participant ":ParserUtil" as Parser LOGIC_COLOR
+end box
+
+[-> MainWindow : executeCommand("help add")
+activate MainWindow
+
+MainWindow -> Parser : parseHelp("help add")
+activate Parser
+Parser --> MainWindow : "Add"
+deactivate Parser
+
+MainWindow -> MainWindow : handleHelp("Add")
+
+create HelpWindow
+MainWindow -> HelpWindow : HelpWindow("Add")
+activate HelpWindow
+HelpWindow -> HelpWindow : setText
+HelpWindow --> MainWindow
+deactivate HelpWindow
+
+MainWindow -> HelpWindow : show()
```

Comment:
Clarification: Is `show()` supposed to return something to the caller? Since theres no return arrow from `:HelpWindow` object